src/pages/blog/[...path].tsx

```diff
@@ -143,6 +142,48 @@ const BlogPage: FC<BlogPageProps> & {
     );
   }, [articles, blogMeta]);
 
+  const getJsonLd = (
+    blogMeta: Nullable<BlogArticleMeta>
+  ): Nullable<BlogArticleJsonLD | object> => {
```

Comment:
The type is a bit weird, object is not a concrete type. I would suggest that we make it cleaner

```suggestion
  const getJsonLd = (
    blogMeta: Nullable<BlogArticleMeta>
  ): Nullable<BlogArticleJsonLD> => {
```

@iamnamananand996 Please refactor function and other related functions to adapt this change